hadoop-tools/hadoop-azure/pom.xml

```diff
@@ -45,6 +45,8 @@
     <fs.azure.scale.test.timeout>7200</fs.azure.scale.test.timeout>
     <fs.azure.scale.test.list.performance.threads>10</fs.azure.scale.test.list.performance.threads>
     <fs.azure.scale.test.list.performance.files>1000</fs.azure.scale.test.list.performance.files>
+    <!-- http connection pool size; passed down as a system property -->
+    <http.maxConnections>100</http.maxConnections>
```

Comment:
Why are we changing the pooling size in JDK from 5 to 100? What about different CPU sizes / network capacity? Would it affect network level metrics like connection establishment latency etc.

Reply:
Hope its for only test runs.